### DISCUSSION TITLE
Changing messages (localizations) of TownyChat

### DISCUSSION THREAD

**Discussion by Milarsh** [TownyAdvanced/Towny#5633] [Q&A] [answered]
Hello! On my server, I want to change the Towny messages. To do this, I changed them in localization, but in the game they remained old. How to fix it?

**Comment by ggvbo** [comment, marked as answer]
Hello,
- Are you modifying the messages by global overriding?
- Did you restart/reload?
- Remember Towny uses your client's locale to display its translations.
